### DISCUSSION TITLE
Failed to open stream: Permission denied

### DISCUSSION THREAD

**Discussion by rogergerberquirino93502** [markshust/docker-magento#780] [General]
Hi, after running command "bin/magento setup:db-declaration:generate-whitelist" I'm getting the following error. Can someone kindly help in resolving thism.                     
                                            
Warning: file_put_contents(/var/www/html/app/code/Mtest/Blog/etc/db_schema_whitelist.json): Failed to open stream: Permission denied in /var/www/html/vendor/magento/framework/Setup/JsonPersistor.php on line 22

**### Resolved**

**Comment by Alekhine-DEV** [comment]
How do you resolve this problem? plz
